### ISSUE TITLE
Remove the command palette from the browser extensions

### ISSUE THREAD

**Issue by philipp-spiess** [sourcegraph/sourcegraph#41114]
For 4.0, we agreed to remove the command palette on web since only extensions could provide implementations for it and without any extensions it would serve no purpose anymore.

We should make sure to do the same in code hosts via the browser extensions when we detect post 4.0 instances.

![Screenshot 2022-08-31 at 12 45 08](https://user-images.githubusercontent.com/458591/187661484-100e74f2-d20f-4858-b782-772e3fb3c3e2.png)

**Comment by philipp-spiess** [comment]
Although hmm there is this one use case that my screenshot shows "Prefer non-Sourcegraph hover tooltips" that still may be worth keeping. Perhaps we should make a button on its own instead? cc @taras-yemets @jjinnii @ryankscott

**Comment by ryankscott** [comment]
@philipp-spiess I guess this is only for our native code host integration? If so, then I agree that this button could be repurposed to toggle the hover behaviour.

**Comment by jjinnii** [comment]
Agreed that rather than keeping the command palette for this one setting, a dedicated action or button for this is preferred. That being said, I don't know if it's as simple as put a button in place of where the command palette icon used to be 😄  We could potentially explore solutions where this setting could be communicated within the popover (upon first use and afterwards). 

I'd like to explore different options, but do we think that this is something we need to support by 4.0? cc: @ryankscott  I also don't know how often users are switching back and forth btw default tooltips. From a product perspective, if a user turns off our browser extension tooltips, what else would they be using the extension for?

**Comment by philipp-spiess** [comment]
Agree, we don't need to make this change before the 4.0 release at all. I'll move this to the post4.0 list. Thanks Sara :)

**Comment by muescha** [comment]
if the command palette shows both tooltips then we need to toggle between.

then an setting in the extension option menu is ok to setup the behaviour:
- show GitHub tooltip
- show SourceGraph hoover tooltip
- show GitHub and ShourceGraph as tooltip
(maybe distinct between when it is shown: `on hover` or `need to click`)


see my Feedback: 
- Chrome extension: show both tooltips: the GitHub and the SourceGraph #44639
